docs/codeql/codeql-language-guides/customizing-library-models-for-javascript.rst

```diff
@@ -431,6 +484,9 @@ The following components are supported:
 - **MapValue** selects a value of a map object.
 - **Awaited** selects the value of a promise.
 - **Instance** selects instances of a class.
+- **Fuzzy** selects all values that are derived from the current value through a combination of the other operations described in this list.
+  For example, this can be used to find all values the appear to originate from a particular package. This can be useful for finding method calls
```

Comment:
```suggestion
  For example, this can be used to find all values that appear to originate from a particular package. This can be useful for finding method calls
```